frontend/public/components/catalog/catalog-item-details.jsx

```diff
@@ -71,6 +87,24 @@ export class CatalogTileDetails extends React.Component {
     const planItems = _.map(plans, (plan) => (
       <li key={plan.metadata.uid}>{plan.spec.description || plan.spec.externalName}</li>
     ));
+    const description = (
+      <>
+        {longDescription && <p>{longDescription}</p>}
+        {sampleRepo && <p>Sample repository: {sampleRepoLink}</p>}
+        {documentationUrl && (
+          <>
+            <h2 className="h5">Documentation</h2>
+            <p>{documentationUrlLink}</p>
+          </>
+        )}
+        {!_.isEmpty(plans) && (
+          <>
+            <h2 className="h5">Service Plans</h2>
+            <ul>{planItems}</ul>
+          </>
+        )}
+      </>
+    );
```

Comment:
IMO, we should include this logic in the `FullDescription` component proposed in my previous comment.

Also, this isn't new, but I would suggest moving the JSX for supportUrlLink, documentationUrlLink, sampleRepoLink, and planItems inline or defining separate React components for them that will handle the null case. The props used to initialize these are not null checked until later, which is a bit risky.